cmd/trust/policygen.go

```diff
@@ -1,54 +1,60 @@
 package main
 
 import (
-	"github.com/project-machine/trust/pkg/trust"
+	"errors"
+
 	"github.com/urfave/cli"
+	"github.com/project-machine/trust/pkg/trust"
 )
 
 var tpmPolicyGenCmd = cli.Command{
 	Name:   "tpm-policy-gen",
-	Usage:  "Generate tpm policy",
+	Usage:  "Generate tpm policy for a keyset",
 	Action: doTpmPolicygen,
+	ArgsUsage: "<keyset-name>",
 	Flags: []cli.Flag{
 		cli.StringFlag{
-			Name:  "pf,passwd-policy-file",
+			Name:  "pf, passwd-policy-file",
 			Usage: "File to which to write password policy",
 			Value: "passwd_policy.out",
 		},
 		cli.StringFlag{
-			Name:  "lf,luks-policy-file",
+			Name:  "lf, luks-policy-file",
 			Usage: "File to which to write luks policy",
 			Value: "luks_policy.out",
 		},
 		cli.StringFlag{
-			Name:  "pp,passwd-pcr7-file",
-			Usage: "File from which to read password pcr7",
-			Value: "passwd_pcr7.bin",
+			Name:  "pcr7-tpm",
+			Usage: "File from which to read uki-tpm pcr7 value",
+			Value: "sha256 pcr7 value",
```

Comment:
I don't think we should create a file with spaces in it by default.

Reply:
Fixed. I also set defaults for output files (the policy digest files).  Also, a default for the policy version since it is a constant right now. I felt the input files (pcr7 files) should be given. I removed the keysetname as an argument. My original intention to include the keysetName was for getting the signing keys. But I decided to make signing a separate thing but forgot to remove keysetname.